language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/extensions/ballerina/document/BallerinaDocumentServiceImpl.java

```diff
@@ -83,28 +83,28 @@ public BallerinaDocumentServiceImpl(LSGlobalContext globalContext) {
     @Override
     public CompletableFuture<BallerinaASTResponse> ast(BallerinaASTRequest request) {
         BallerinaASTResponse reply = new BallerinaASTResponse();
-        String fileUri = request.getDocumentIdentifier().getUri();
```

Comment:
@nadeeshaan is this intentional? we are commenting extension service method, wouldn't this affect the other external teams?

Reply:
Yes, this API is not effectively used by the other extension clients and since they have to migrate to the latest ProjectAPI this will not be working on top of the old LSCompiler. Hence we disabled this to avoid unnecessary NPEs thrown